plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -583,6 +583,12 @@ private void invalidateColumnsCache(SchemaTableName table)
         invalidateCache(columnsCache, key -> key.table.equals(table));
     }
 
+    @VisibleForTesting
+    CacheStats getSchemaNamesCacheStats()
```

Comment:
We can split this commit into two - First we could have a commit which `Expose SchemName cache stats` which expose the stats and a corresponding testing. Then a commit which extracts table handler assertions. WDYT ?